contracts/test/FixedPointTest.sol

```diff
@@ -22,7 +22,7 @@ contract FixedPointTest {
         return FixedPoint.decode144(self);
     }
 
-    function mul(FixedPoint.uq112x112 calldata self, uint256 y) external pure returns (FixedPoint.uq144x112 memory) {
+    function mul(FixedPoint.uq112x112 calldata self, uint32 y) external pure returns (FixedPoint.uq144x112 memory) {
```

Comment:
hmm can we just optimize the < uint32 case?

Reply:
yeah, i think we might not need this at all though, maybe worth just removing a lot of these unused functions